### PR TITLE
LOC CHECKIN | dotnet/roslyn master-vs-deps | 20180629

### PR DESCRIPTION
**Pull request by v-zbsail**

### CODE REVIEW

File: src/Features/Core/Portable/xlf/FeaturesResources.de.xlf [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <body>
       <trans-unit id="Add_parameter_to_0_and_overrides_implementations">
         <source>Add parameter to '{0}' (and overrides/implementations)</source>
-        <target state="new">Add parameter to '{0}' (and overrides/implementations)</target>
+        <target state="translated">Parameter zu "{0}" (und Außerkraftsetzungen/Implementierungen) hinzufügen</target>
```
Comment (jhinder) [inline src/Features/Core/Portable/xlf/FeaturesResources.de.xlf]: "Außerkraftsetzungen" is inconsistent with the rest of the German translation. In other places, "Überschreibungen" is used as the translation for the noun "overrides" (cf. line [1785 in `FeaturesResources.de.xlf`](https://github.com/dotnet/roslyn/blob/master/src/Features/Core/Portable/xlf/FeaturesResources.de.xlf#L1785), line [1477 in `CSharpResources.de.xlf`](https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Portable/xlf/CSharpResources.de.xlf#L1477)).

File: src/VisualStudio/CSharp/Impl/xlf/CSharpVSResources.de.xlf [hunk 2]
```diff
@@ -4,47 +4,47 @@
...
       <trans-unit id="Add_remove_braces_for_single_line_control_statements">
         <source>Add/remove braces for single-line control statements</source>
-        <target state="new">Add/remove braces for single-line control statements</target>
+        <target state="translated">Geschweifte Klammern für einzeilige Steuerungsanweisungen hinzufügen/entfernen</target>
         <note />
       </trans-unit>
       <trans-unit id="Apply_all_csharp_formatting_rules_indentation_wrapping_spacing">
         <source>Apply all C# formatting rules (indentation, wrapping, spacing)</source>
-        <target state="new">Apply all C# formatting rules (indentation, wrapping, spacing)</target>
+        <target state="translated">Alle C#-Formatierungsregeln anwenden (Einzug, Umbruch, Abstand)</target>
```
Comment (jhinder) [inline src/VisualStudio/CSharp/Impl/xlf/CSharpVSResources.de.xlf]: "Einzug" is used in typography and printing, "Einrückung" is used in programming. (See also: line [72 in `BasicVSResources.de.xlf`](https://github.com/dotnet/roslyn/blob/6c7c773ae6ea0ac85951e4c83c33db2ef6b3dd16/src/VisualStudio/VisualBasic/Impl/xlf/BasicVSResources.de.xlf#L72), the [German Wikipedia article on "Indentation style"](https://de.wikipedia.org/wiki/Einr%C3%BCckungsstil).)

Comment (janbaltus) [reply src/VisualStudio/CSharp/Impl/xlf/CSharpVSResources.de.xlf]: Thank you for your feedback. We will update transaltion during next round of localization

File: src/Compilers/CSharp/Portable/xlf/CSharpResources.ja.xlf [hunk 3]
```diff
@@ -4,7 +4,7 @@
     <body>
       <trans-unit id="ERR_OutVariableCannotBeByRef">
         <source>An out variable cannot be declared as a ref local</source>
-        <target state="translated">out 変数を ref ローカルと宣言することはできません</target>
+        <target state="translated"></target>
```
Comment (tmeschter) [inline src/Compilers/CSharp/Portable/xlf/CSharpResources.ja.xlf]: The translation seems to have been dropped entirely.

**Comment by v-zbsail** [comment]
Hello Tom,

I’ve committed the fix for this.

Best regards,

Zbyněk Sailer | Logistics IPE
T: +420-545-552-396 | M: +420-731-450-569
v-zbsail@microsoft.com<mailto:v-zbsail@microsoft.com> | zsailer@moravia.com<mailto:zsailer@moravia.com> | moravia.com<http://www.moravia.com/>

**Comment by tmeschter** [comment]
test this

**Comment by tmeschter** [comment]
test this please

**Comment by tmeschter** [comment]
This has been supplanted by more recent fixes.

**Pull request closed without merging**